### ISSUE TITLE
datepicker today date is incorrect

### ISSUE THREAD

**Issue by danielcollettepb** [PBGUX/designsystem#132]
was looking at the DS date pickers and found another bug. I  was looking at the control that shows only one date if user is looking at one day range (or today) . What I found was that while it works great for Today, the word today is not changing when I picked another one date as range. It think there should be statement added that id date is not today, the word “Today” should not be displayed. In example below you will see that I picked May 2 and it still says Today. 
Thanks,
Ania

![image](https://cloud.githubusercontent.com/assets/6116867/15338676/ede4c132-1c4e-11e6-8581-8cfe7c29c4c6.png)

**Comment by danielcollettepb** [comment]
This is done by design in the code and can be changed in your implementation, this is not a bug.
 
We compare the dates and set the display to ‘Today’ if the start and end dates match (i.e. 0 difference)
 
```
if (dateDiff === 0) { //if  the start and end date match (i.e. zero difference)
          result = 'Today ' + moment(startDate).format('MM/DD/YYYY'); // then display as ‘Today’
        } else if (dateDiff === 6) {
          result = 'Last 7 days';
        } else {
          result = 'From ' + moment(startDate).format('MM/DD/YYYY') + ' to ' + moment(endDate).format('MM/DD/YYYY');
        }
 ```

Point is, the control has lots of configurations that the team can implement based on what you want.
 
If I remember correct, Ron/joan didn’t want the same date displayed twice in the from and to since it’s not a range but a single date.

**Comment by danielcollettepb** [comment]
I agree that this one date is a nicer thing to have instead of same date displayed twice, What I am saying is that “Today" should only be display if the date is actually today’s date and not 2 weeks ago for example. In that case it should only display the date and not word “Today” with it.

**Comment by danielcollettepb** [comment]
Ahh, that’s clearer now, thanks.

**Comment by danielcollettepb** [comment]
fixed in 2.8
